src/main/java/easy/mind/com/api/service/Impl/BeckDepressionInventoryServiceImpl.java

```diff
@@ -37,7 +36,9 @@ public BeckDepressionInventoryDTO create(BeckDepressionInventoryDTO inventoryDTO
 
     @Override
     public List<BeckDepressionInventoryDTO> getByUserId(int userId) {
-        UserDTO user = userService.readById(userId);
+        if (userService.throwIfNotExist(userId)) {
```

Comment:
It's confusing, method named as 'throw' but it throws here. Do this check and throw in userService. Here it should be simple call `userService.throwIfNotExists(userId)`, `userService.validateExistence(userId)`, etc...